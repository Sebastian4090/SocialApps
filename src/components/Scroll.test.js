import { shallow } from 'enzyme';
import React from 'react';
import Scroll from './Scroll';

it('expect to render Scroll component', () => {
    expect(shallow(<Scroll />)).toMatchSnapshot()
})