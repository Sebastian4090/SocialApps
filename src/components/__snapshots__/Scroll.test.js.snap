// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Scroll component 1`] = `
<div
  style={
    Object {
      "border": "2px solid black",
      "height": "800px",
      "overflowY": "scroll",
    }
  }
/>
`;
